src/main/kotlin/no/nav/syfo/sykmeldingstatus/SykmeldingStatusService.kt

```diff
@@ -148,6 +151,54 @@ class SykmeldingStatusService(
         )
     }
 
+    suspend fun finnTidligereArbeidsgivere(
+        fnr: String,
+        sykmeldingId: String
+    ): List<TidligereArbeidsgiver>? {
+        val sykmeldingerWithStatus = sykmeldingStatusDb.getSykmeldingWithStatus(fnr)
+        val sykmeldingWithStatus = sykmeldingerWithStatus.find { it.sykmeldingId == sykmeldingId }
+        if (sykmeldingWithStatus != null) {
```

Comment:
early exit isteden for å wrappe hele happy-casen i en blokk

Reply:
👍